site/layouts/partials/header.html

```diff
@@ -11,6 +11,8 @@
 
 <link rel="canonical" href="{{ .Permalink }}">
 
+<link rel="preconnect" href="https://AK7KMZKZHQ-dsn.algolia.net" crossorigin>
```

Comment:
This should have the same check as the CSS. It doesn't make sense to preload this file otherwise.

Reply:
BTW where does this file comes from? Is it documented somewhere?